package/gluon-config-mode-geo-location/files/lib/gluon/config-mode/wizard/0400-geo-location.lua

```diff
@@ -1,9 +1,20 @@
 local cbi = require "luci.cbi"
 local i18n = require "luci.i18n"
 local uci = luci.model.uci.cursor()
+local site = require 'gluon.site_config'
 
 local M = {}
 
+function show_altitude()
+  if site.config_mode == nil or site.config_mode.geo_location == nil or site.config_mode.geo_location.show_altitude == nil or site.config_mode.geo_location.show_altitude == true then
```

Comment:
This could be written a lot shorter as:

    if ((site.config_mode or {}).geo_location or {}).show_altitude ~= false then